source/MaterialXFormat/Util.cpp

```diff
@@ -14,6 +14,31 @@
 namespace MaterialX
 {
 
+namespace
+{
+
+FileSearchPath getSubDirectories(const FileSearchPath paths)
```

Comment:
Nitpick: I don't "think" an incorrect call can occur just to avoid using the same name as the utility maybe call this getUniqueSubDirectories() or something like that ?